<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>795</width>
    <height>642</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>795</width>
    <height>642</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>795</width>
    <height>642</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>计算机图形学个人作业</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>90</y>
      <width>131</width>
      <height>471</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="drawLine">
         <property name="text">
          <string>直线</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="drawCircle">
         <property name="text">
          <string>圆</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="drawEllipse">
         <property name="text">
          <string>椭圆</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="fillArea">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>区域填充</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="BezierSpline">
         <property name="text">
          <string>Bezier曲线</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="B_Spline">
         <property name="text">
          <string>B-样条曲线</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="Koch">
         <property name="text">
          <string>Koch曲线</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Mandelbrot">
         <property name="text">
          <string>Mandelbrot集</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Julia">
         <property name="text">
          <string>Julia集</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="fern">
         <property name="text">
          <string>蕨类植物</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>0</y>
      <width>133</width>
      <height>68</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>3</horstretch>
         <verstretch>5</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>17</pointsize>
        </font>
       </property>
       <property name="text">
        <string>张家强</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>17</pointsize>
        </font>
       </property>
       <property name="text">
        <string>SC17023010</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>795</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <action name="line">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>直线</string>
   </property>
  </action>
  <action name="circle">
   <property name="text">
    <string>圆</string>
   </property>
  </action>
  <action name="curve">
   <property name="text">
    <string>曲线</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>区域填充</string>
   </property>
  </action>
  <action name="actionBezier">
   <property name="text">
    <string>Bezier曲线</string>
   </property>
  </action>
  <action name="actionB">
   <property name="text">
    <string>B-样条曲线</string>
   </property>
  </action>
  <action name="actionKoch">
   <property name="text">
    <string>Koch曲线</string>
   </property>
  </action>
  <action name="actionMandelbrot">
   <property name="text">
    <string>Mandelbrot集</string>
   </property>
  </action>
  <action name="actionJulia">
   <property name="text">
    <string>Julia集</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>蕨类植物</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
